File: templates/python/v/latest/src/task.py
import os

def task_handler(event):
    msg = f"Hello from {os.getenv('RUNTIME_ENV', 'unknown')}!"
    print(msg)

    # Print the event arguments
    print("Received event:", event)

    response = {
        "statusCode": 200,
        "body": msg
    }

    return response

File: templates/nodejs/v/latest/src/task.js
export const handler = async (event, context) => {
  const runtimeEnv = process.env.RUNTIME_ENV || 'unknown';
  const msg = `Hello from ${runtimeEnv}!`;
  console.log(msg);

  // Print the received event and context
  console.log('Received event:', event);
  console.log('Received context:', context);

  const response = {
    statusCode: 200,
    body: JSON.stringify(msg),
  };

  return response;
};
